abpframework/eShopOnAbp
# Account management and User management issues

Fixes #176

This PR adds one-way-synchronization from Identity management to the Keycloak server. 

Whenever a new user or role being created,updated or deleted; it will be synched to the Keycloak server.
It uses background jobs and write-through caching by overriding the existing Identity and the Role management.
`Keycloak.Net.Core` package is being used for keycloak interactions.

### Testing Steps
@gizemmutukurt If you can find time to test;
Run the solution using `run-tye` command.
Run the back-office app by running `yarn start` under `apps/angular` folder.
- Add new roles, users
- Update roles, users

Observe changes at keycloak server (localhost:8080) under Realm Roles and Users, user details sections.
